docs/docs/advanced/air-gap.md

```diff
@@ -157,6 +105,43 @@ For Java DB the process is the same, except for the following:
 
 ## Misconfigurations scanning
 
-Note that the misconfigurations checks bundle is also embedded in the Trivy binary (at build time), and will be used as a fallback if the external database is not available. This means that you can still scan for misconfigurations in an air-gapped environment using the Checks from the time of the Trivy release you are using.
+### Connectivity requirements
+
+Trivy's misconfiguration database is packaged as an OCI image and follows the same connectivity requirements as the Vulnerability and Java databases, as can be seen [here](#vulnerability-java-databases).
+
+### Self-hosting
+
+You can host Trivy's databases in your own container registry. Please refer to [Self-hosting document](./self-hosting.md) for a detailed guide.
+
+### Embedded checks
+
+Misconfigurations checks are embedded in the Trivy binary (at build time), and will be used as a fallback if the external database is not available. This means that you can still scan for misconfigurations in an air-gapped environment using the Checks from the time of the Trivy release you are using.
```

Comment:
ditto

```suggestion
Misconfigurations database is embedded in the Trivy binary (at build time), and will be used as a fallback if the external database is not available. This means that you can still scan for misconfigurations in an air-gapped environment using the database from the time of the Trivy release you are using.
```

Reply:
my implicit thought process was the checks from the misconfiguration db are embedded as a workaround to avoid loading the db, but it's very semantic and doesn't really matter.

Reply:
I think we should define terms regarding misconfigurations. What is the misconfiguration database? Is it pointing to the bundle.tar.gz? After extracting it, do we call it checks? Embedded one is not archived, so we call it checks?

I don't have an opinion, but it would be a good idea to define the terminology and share it within the project.
@simar7 @nikpivkin

Reply:
I don't think it's resolved. I want to make a decision on terminology excplicitly. Otherwise, "check", "policy" and "database" will be mixed up in the document in the future. Also, as I mentioned, we need to rename the CLI flag of `--checks-bundle` if we change the term.
@simar7 @nikpivkin 

Also, there are several places calling it "checks bundle".
https://trivy.dev/v0.57/docs/scanner/misconfiguration/check/builtin/#checks-bundle
https://trivy.dev/v0.57/docs/references/modes/client-server/#architecture

Reply:
I discussed with AI and now would like to vote for "bundle" as below:

- check: Rego file
- built-in checks: Rego files from trivy-checks
- custom checks: user-defined Rego files
- checks bundle: bundle.tar.gz from trivy-checks
- embedded checks: Rego files embedded into the binary

But I just want to make them consistent in the project. Any others are also fine for me.

Reply:
This terminology list makes sense to me.. But I think we already use it this way so not sure what is the change you are suggesting
1. policy -> check: was discussed before (I can find the issue)
2. built-in/custom/embedded: it only makes sense to contextualize when needed, I think we use this terminology this way already.
3. bundle: this is OPA terminology that I believe we borrowd which is fine, when applied to the distributable tar.gz.

I'm trying to clarify your feedback, is it against using the word "database" for checks? If so, the list doesn't address this case, what is your suggstion?

Reply:
Yes, I believe this is how we use terms now, but I wanted to ensure we're on the same page. And then, 

1. Do we all agree to add the term "database"?
2. If so, how will we update the terminology?

Reply:
@knqyf263 specially for the suggestion that you made above, I'm not sure if we need to change the terminology to use "database".

What you mentioned [here](https://github.com/aquasecurity/trivy/pull/7732#discussion_r1856423997) makes sense to me and is the current terminology that we agree on.

If so, is there any action item that still needs to addressed out of this?

Reply:
This PR introduces several new terms, like [Misconfigurations Database](https://github.com/aquasecurity/trivy/pull/7732/files#diff-f67fbe532cb6f906f5e2e056c318da6d0bc245f94c8cc1afbd7e72a61e2506b8R12), [Embedded misconfiguration database](https://github.com/aquasecurity/trivy/pull/7732/files#diff-f67fbe532cb6f906f5e2e056c318da6d0bc245f94c8cc1afbd7e72a61e2506b8R42) and [Checks Database](https://github.com/aquasecurity/trivy/pull/7732/files#diff-7ad34887030b99ec320003195c3655f39b642b88ca996022ba52ee273c075ffbR73). I want to clarify what it refers to and whether we agree on the term. If we prefer [the current terminology](https://github.com/aquasecurity/trivy/pull/7732#discussion_r1856423997), we should delete "misconfigurations database" or things like that from this PR.

Reply:
how should we call it then?

Reply:
I don't probably understand your suggestion. What is "misconfigurations database"?

I thought it was trying to replace "checks bundle", so according to the conclusion that we'd stay with the current terminology, we continue to use "checks bundle". But I'm confused with your question. Does "misconfigurations database" mean something new?

Reply:
ok, I thought that we call all the Trivy's native sources of information - databases, regardless of their underlying technology or file format. I thought bundle is just a technical term we inharited from OPA to refer to the file format (specified by OPA), but we still call it Checks Database when referring to it's functionality in Trivy, so I didn't view those terms as mutually exclusive. I don't mind calling it Checks Bundle everywhere, if that's what you suggest (but I also think it's not benefiting the user experience). Do you also think we should remove it from the databases page?

Reply:
>but we still call it Checks Database when referring to it's functionality in Trivy

Oh, I didn't know that. I thought "Checks Database" was introduced in this PR. Where do we use the term in Trivy? 

Since we already use "Checks Bundle" in the documentation and have a CLI flag "--checks-bundle-repository", it's user-facing. We should use either "Checks Bundle" or "Checks Database". If we go for "Checks Database", we should delete "Checks Bundle" and vice versa.

BTW, I'd propose calling the external data "Scan Assets" instead of "Databases". Then, it's natural to keep "checks bundle" under the page.
https://github.com/aquasecurity/trivy/pull/7996

Reply:
>I wrote that I thought we still call it Checks Database.. probably my misunderstanding

Hmm. @simar7 and @nikpivkin should be more familiar with that. I've personally never seen that.

>do you consider VEX Hub and/or maven central as scan assets? how would you refer to (for example) the VEX Hub file (not a database I suppose)?

I defined "Scan Assets" as external data that Trivy downloads and uses during scanning.
https://github.com/aquasecurity/trivy/pull/7996/files#diff-cca74cb5f95c26518039a73ea36336112824080a13e672d69372083b5a4ad7f7R19

Therefore, VEX Hub is an asset as Trivy downloads it during scanning, but any API call, like Maven Central, is not an asset.
However, I'd put this discussion aside. I'll open a follow-up PR to share my suggestion. We can discuss "Checks Database" or "Checks Bundle" here.

Reply:
I thought "misconfiguration database" was a newly introduced term. @itaysk thought "misconfigurations database" was already used and just reused it.

Most pages call it a checks bundle. So, a checks bundle is the canonical term, IMO. I think we should comply with that.

Reply:
so next steps to merge this PR - 
1. change Checks DB to checks bundle
2. change Databases page to scan assets
does that make sense to you?

Reply:
I'm fine with only the first item as the second item requires additional changes. Let me try the second one in a separate PR.

Reply:
lgtm